libpod/pod.go

```diff
@@ -96,7 +97,8 @@ type PodContainerInfo struct {
 
 // InfraContainerConfig is the configuration for the pod's infra container
 type InfraContainerConfig struct {
-	HasInfraContainer bool `json:"makeInfraContainer"`
+	HasInfraContainer bool                           `json:"makeInfraContainer"`
+	PortBindings      map[nat.Port][]nat.PortBinding `json:"infraPortBindings"`
```

Comment:
Can we make these match the CNI structs that Libpod uses? So we can move the heavy lifting around ports into cmd/podman and turn CreatePortBindings into a straight assignment when we make the pod container